assets/xml/objects/object_zov.xml

```diff
@@ -1,81 +1,82 @@
-        <Limb Name="object_zov_Standardlimb_016140" Type="Standard" EnumName="OBJECT_ZOV_LIMB_07" Offset="0x16140" />
-        <Limb Name="object_zov_Standardlimb_01614C" Type="Standard" EnumName="OBJECT_ZOV_LIMB_08" Offset="0x1614C" />
-        <Limb Name="object_zov_Standardlimb_016158" Type="Standard" EnumName="OBJECT_ZOV_LIMB_09" Offset="0x16158" />
-        <Limb Name="object_zov_Standardlimb_016164" Type="Standard" EnumName="OBJECT_ZOV_LIMB_0A" Offset="0x16164" />
-        <Limb Name="object_zov_Standardlimb_016170" Type="Standard" EnumName="OBJECT_ZOV_LIMB_0B" Offset="0x16170" />
-        <Limb Name="object_zov_Standardlimb_01617C" Type="Standard" EnumName="OBJECT_ZOV_LIMB_0C" Offset="0x1617C" />
-        <Limb Name="object_zov_Standardlimb_016188" Type="Standard" EnumName="OBJECT_ZOV_LIMB_0D" Offset="0x16188" />
-        <Limb Name="object_zov_Standardlimb_016194" Type="Standard" EnumName="OBJECT_ZOV_LIMB_0E" Offset="0x16194" />
-        <Limb Name="object_zov_Standardlimb_0161A0" Type="Standard" EnumName="OBJECT_ZOV_LIMB_0F" Offset="0x161A0" />
-        <Limb Name="object_zov_Standardlimb_0161AC" Type="Standard" EnumName="OBJECT_ZOV_LIMB_10" Offset="0x161AC" />
-        <Limb Name="object_zov_Standardlimb_0161B8" Type="Standard" EnumName="OBJECT_ZOV_LIMB_11" Offset="0x161B8" />
-        <Limb Name="object_zov_Standardlimb_0161C4" Type="Standard" EnumName="OBJECT_ZOV_LIMB_12" Offset="0x161C4" />
-        <Limb Name="object_zov_Standardlimb_0161D0" Type="Standard" EnumName="OBJECT_ZOV_LIMB_13" Offset="0x161D0" />
-        <Limb Name="object_zov_Standardlimb_0161DC" Type="Standard" EnumName="OBJECT_ZOV_LIMB_14" Offset="0x161DC" />
-        <Limb Name="object_zov_Standardlimb_0161E8" Type="Standard" EnumName="OBJECT_ZOV_LIMB_15" Offset="0x161E8" />
-        <Limb Name="object_zov_Standardlimb_0161F4" Type="Standard" EnumName="OBJECT_ZOV_LIMB_16" Offset="0x161F4" />
-        <Skeleton Name="object_zov_Skel_016258" Type="Flex" LimbType="Standard" LimbNone="OBJECT_ZOV_LIMB_NONE" LimbMax="OBJECT_ZOV_LIMB_MAX" EnumName="ObjectZovLimb" Offset="0x16258" />
+        <Animation Name="gLuluTurnAndWalkAnim" Offset="0x17D4" />
+        <Animation Name="gLuluWalkLoopAnim" Offset="0x23F4" />
+        <Animation Name="gLuluSingStartAnim" Offset="0x2B5C" />
+        <Animation Name="gLuluSingLoopAnim" Offset="0x418C" />
+        <Animation Name="gLuluLookForwardAndLeftAnim" Offset="0x5A6C" />
+        <Animation Name="gLuluLookLeftLoopAnim" Offset="0x66A4" />
+        <Animation Name="gLuluPutHandsDownAnim" Offset="0x8120" />
+        <Animation Name="gLuluLookAroundAnim" Offset="0xA888" />
+        <Animation Name="gLuluLookForwardAndDownAnim" Offset="0xB4CC" />
+        <Animation Name="gLuluAngleHeadAnim" Offset="0xC510" />
+        <Animation Name="gLuluNodAnim" Offset="0xCAA8" />
+        <Animation Name="gLuluLookDownAnim" Offset="0xD3EC" />
+        <DList Name="gLuluTorsoDL" Offset="0x107B0" />
```

Comment:
```suggestion
        <Animation Name="gLuluLookDownAnim" Offset="0xD3EC" />
        
        <!-- Lulu Limb DisplayLists -->
        <DList Name="gLuluTorsoDL" Offset="0x107B0" />
```
Just one minor comment. Could you break up the sections like this? Just to help with readability